[Merged by Bors] - Port tests from original java implementation

ozgrakkurt opened the pull request:
closes https://github.com/infinyon/fluvio-jolt/issues/19

sehz commented on tests/java/resources/cardinality/atTestData.json:
```diff
@@ -0,0 +1,105 @@
+{
```
can you write description and link where this came from for all data copies from other repo or put these data into separate directory and put README

ozgrakkurt replied:
can I add them as a git submodule?

sehz replied:
Sure

ozgrakkurt replied:
done

sehz reviewed (approved):
LGTM

ozgrakkurt commented:
bors r+

galibey commented:
Are these tests failing? If so, why we introduced failed tests into the repo? 

The CI should have run all tests, but it runs only unit tests `--lib`.

ozgrakkurt commented:
> Are these tests failing? If so, why we introduced failed tests into the repo?
> 
> The CI should have run all tests, but it runs only unit tests `--lib`.

Yes, I didn't want to do all changes in a single PR so I added these here and I'll do the implementations in the next and can enable integration tests on CI then. Integration tests were already disabled in CI so no regression on this PR.

galibey commented:
All tests should be green for the local setup as well, not only CI. If there is an activity in progress, tests should be ignored.
